components/scream/src/physics/shoc/tests/shoc_varorcovar_tests.cpp

```diff
@@ -270,6 +270,36 @@ struct UnitWrap::UnitTest<D>::TestShocVarorCovar {
     }
   }
 
+
+static void run_bfb()
+  {
+    SHOCVarorcovarData SDS_f90[] = {
+      //               shcol, nlev, nlevi, tunefac
+      SHOCVarorcovarData(10, 71, 72, 1),
+      SHOCVarorcovarData(10, 12, 13, 1),
+      SHOCVarorcovarData(7,  16, 17, 1),
+      SHOCVarorcovarData(2, 7, 8, 0.005),
+    };
+
+    for (auto& d_f90 : SDS_f90) {
+      // Randomize remaining inputs
+      d_f90.randomize();
+
+      // Get data from fortran, expects data in C layout
+      calc_shoc_varorcovar(d_f90);
+
+      // Get data from cxx, expects data in fortran layout
+      SHOCVarorcovarData d_cxx = SHOCVarorcovarData(d_f90);
```

Comment:
An explicit copy construction would be cleaner here.

Also, we should not get into the habit of doing the cxx Data copy after retrieving the f90 data. This will break if there is inout data.